transport.rb

```diff
@@ -0,0 +1,45 @@
+class Transport
+
+  include Comparable , Enumerable
+
+  attr_accessor :current_weight , :available
+
+  CONVERT_TO_MINUTES = 60 
+
+  def initialize(current_weight)
+    @current_weight = current_weight
+    @available = true
+  end
+  
+  def max_weight
+    raise NotImplementedError
+  end
+
+  def max_weight=(value)
+    raise NotImplementedError
+  end
+
+  def speed
+    raise NotImplementedError
+  end
+
+  def speed=(value)
+    raise NotImplementedError
+  end
```

Comment:
it is over engineering.
[Here](https://github.com/Andy1391/delivery_service/pull/1/commits/a1d95417abeb946549104048a3c4e684b157b4af#r410218562) I proposed to raise it at `initialize` method

Reply:
Done